extensions/txAdminClient/resource/fxmanifest.lua

```diff
@@ -1,4 +1,6 @@
-resource_manifest_version '44febabe-d386-4d18-afbe-5e627f4af937'
+fx_version 'adamant'
```

Comment:
Now, we are at 'bodacious'

[source](https://docs.fivem.net/docs/scripting-manual/introduction/introduction-to-resources/)